modules/apps/app-builder/app-builder-test/src/testFunctional/macros/AppBuilderAdmin.macro

```diff
@@ -174,6 +188,16 @@ definition {
 			locator1 = "AppBuilder#CHECKBOX_OPTION_LABEL");
 	}
 
+	macro undeployApp {
+		Click(
+			key_itemName = "${itemName}",
+			locator1 = "AppBuilder#VERTICAL_ELLIPSIS_BUTTON");
+
+		MenuItem.clickNoError(menuItem = "Undeploy");
+
+		Click(locator1 = "AppBuilder#UNDEPLOY_APP_BUTTON");
+	}
+
```

Comment:
Hi @nkkwon ,
I was wondering here if we should have to validate the table status "Undeployed" as well, similar to what you are doing at the deleteApp, asserting that the element is no longer present)

What do you think?

@alexcandido10